lib/sisimai/bite/email/exim.rb

```diff
@@ -0,0 +1,522 @@
+              #
+              # deliver.c:4549|  printed = US"an undisclosed address";
+              #   an undisclosed address
+              #     (generated from kijitora@example.jp)
+              if v['recipient']
+                # There are multiple recipient addresses in the message body.
+                dscontents << Sisimai::Bite.DELIVERYSTATUS
+                v = dscontents[-1]
+              end
+              v['recipient'] = cv[1]
+              recipients += 1
+
+            elsif cv = e.match(/\A[ ]+[(]generated[ ]from[ ](.+)[)]\z/) ||
+                       e.match(/\A[ ]+generated[ ]by[ ]([^ \t]+[@][^ \t]+)/)
+              #     (generated from kijitora@example.jp)
+              #  pipe to |/bin/echo "Some pipe output"
+              #    generated by userx@myhost.test.ex
+              v['alias'] = cv[1]
+
+            else
+              next if e.empty?
+
+              if e =~ Re1[:frozen]
+                # Message *** has been frozen by the system filter.
+                # Message *** was frozen on arrival by ACL.
+                v['alterrors'] ||= ''
+                v['alterrors']  += e + ' '
+
+              else
+                if boundary00.size > 0
+                  # --NNNNNNNNNN-eximdsn-MMMMMMMMMM
+                  # Content-type: message/delivery-status
+                  # ...
+                  if cv = e.match(/\A[Rr]eporting-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                    # Reporting-MTA: dns; mx.example.jp
+                    v['lhost'] = cv[1]
+
+                  elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                    # Action: failed
+                    v['action'] = cv[1].downcase
+
+                  elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                    # Status: 5.0.0
+                    v['status'] = cv[1]
+
+                  elsif cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                    # Diagnostic-Code: SMTP; 550 5.1.1 <userunknown@example.jp>... User Unknown
+                    v['spec'] = cv[1].upcase
+                    v['diagnosis'] = cv[2]
+
+                  elsif cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*(.+)\z/)
+                    # Final-Recipient: rfc822;|/bin/echo "Some pipe output"
+                    v['spec'] ||= cv[1] =~ /[@]/ ? 'SMTP' : 'X-UNIX'
+
+                  else
+                    # Error message ?
+                    if havepassed[:deliverystatus].zero?
+                      # Content-type: message/delivery-status
+                      havepassed[:deliverystatus] = 1 if e =~ Re1[:deliverystatus]
+                      v['alterrors'] ||= ''
+                      v['alterrors']  += e + ' ' if e =~ /\A[ ]+/
+                    end
+                  end
+                else
+                  if dscontents.size == recipients
+                    # Error message
+                    next unless e.size
+                    v['diagnosis'] ||= ''
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->